src/nvim/fileio.h

```diff
@@ -18,7 +18,7 @@
 #define READ_NOWINENTER 0x80    // do not trigger BufWinEnter
 #define READ_NOFILE     0x100   // do not read a file, do trigger BufReadCmd
 
-#define READ_STRING(x, y) (char_u *)read_string((x), (size_t)(y))
+#define READ_STRING(x, y) read_string((x), (size_t)(y))
```

Comment:
This macro can be removed. It's not present in Vim either.

Reply:
I get an error when I try to remove it. It seems to be used in spellfile.c.

Reply:
Oh you mean just substitute the macro, gotcha.